vendor/bat-native-ads/src/bat/ads/internal/ml/data/vector_data_aliases.h

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ML_DATA_VECTOR_DATA_ALIASES_H_
+#define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_ML_DATA_VECTOR_DATA_ALIASES_H_
+
+#include <cstdint>
```

Comment:
for consistency in ads can we have always used `#include <stdint.h>`, however after checking Chromium coding style we should now use `#include <cstdint>`. I have raised https://github.com/brave/brave-browser/issues/14446 to fix this across Brave ads

Reply:
Great!